.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12.10.0
+    steps:
+      - checkout
+      - run:
+          name: Install Dependencies
+          command: npm install
+      - run:
+          name: Run Linter
+          command: npm run lint
+      - run:
+          name: Compile
+          command: npm run dev
+      - run:
```

Comment:
You can set  `shell: /bin/sh`.

https://circleci.com/docs/2.0/configuration-reference/#default-shell-options